File: medplum-typescript/packages/sdk/src/model/fhir/ExampleScenario_Alternative.ts
import { ExampleScenario_Step } from "./ExampleScenario_Step";
import { Extension } from "./Extension";
import { markdown } from "./markdown";

export interface ExampleScenario_Alternative {
    id: string | null | undefined;
    extension: Extension[] | null | undefined;
    modifierExtension: Extension[] | null | undefined;
    title: string | null | undefined;
    description: markdown | null | undefined;
    step: ExampleScenario_Step[] | null | undefined;
}

File: medplum-java/medplum-model/src/main/java/com/fhir/ExampleScenario_Alternative.java
package com.fhir;

import com.StagedBuilderStyle;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import java.lang.String;
import java.util.List;
import java.util.Optional;
import org.immutables.value.Value;

@Value.Immutable
@StagedBuilderStyle
@JsonDeserialize(
    as = ImmutableExampleScenario_Alternative.class
)
@JsonIgnoreProperties(
    ignoreUnknown = true
)
public interface ExampleScenario_Alternative {
  Optional<List<Extension>> modifierExtension();

  Optional<List<ExampleScenario_Step>> step();

  Optional<String> id();

  Optional<List<Extension>> extension();

  Optional<String> title();

  Optional<markdown> description();

  static ImmutableExampleScenario_Alternative.Builder builder() {
    return ImmutableExampleScenario_Alternative.builder();
  }
}
